.ci/scripts/conformance-scripts/patcher_of_tests.py

```diff
@@ -0,0 +1,44 @@
+import re
+import sys
+
+alg_name = sys.argv[1]
+
+testToPatch = ['dbscan','elastic_net', 'lin_reg', 'ridge_reg']
+
+case = {
```

Comment:
Why can't I see cases for `svm`, `kmeans` and `pca` ?

Reply:
In tests for some algorithms import going from module, but for other import going from specific file. This patch needed for this case.